draft-annevk-johannhof-httpbis-cookies.md

```diff
@@ -1193,11 +1227,30 @@ boolean _httpOnlyAllowed_, boolean _allowNonHostOnlyCookieForPublicSuffix_, and
 
 1. Insert _cookie_ into the user agent's cookie store.
 
-1. Remove all expired cookies from the user agent's cookie store.
+1. Return _cookie_.
+
+
+### Garbage Collect Cookies
+
+To **Garbage Collect Cookies** given a host _host_:
```

Comment:
Allowing user agents to GC all cookies periodically could be done with this and a hack (_host_ = new opaque host) but it would be better to make the argument optional here. Is that possible?

Reply:
We talked offline and it seems like we don't need this right now if you don't want to touch it

Reply:
The rationale is that we don't actually know any callers that would want to call this without host. So until we identify such a caller we shouldn't make a change here.